include/seqan/basic/iterator_adaptor.h

```diff
@@ -96,10 +96,10 @@ class Iter<TContainer, AdaptorIterator<TIterator, TSpec> >
     // Constructors
     // ------------------------------------------------------------------------
 
-   
-    Iter() : data_container()
+
+    Iter() : data_container(),
+             data_iterator()
     {
```

Comment:
put the closing bracket on the same line if content is empty

Reply:
The other constructors also put the closing bracket on the next line 🤔

Reply:
True, but this does not follow the seqan [style guides](https://seqan.readthedocs.io/en/master/Infrastructure/Contribute/StyleCpp.html#constructor-initialization-lists) so when you touch the code I think it's better to do it "the right way"?